sdk/core/src/CommonNetworkMember.ts

```diff
@@ -1992,20 +1982,49 @@ export abstract class CommonNetworkMember<TOptions extends SdkOptions = SdkOptio
   }
 
   /**
-   * @description Retrieve only the credential at given index
-   * @param credentialIndex - index for the VC in vault
+   * @description Retrieve only the credential
+   * @param credentialId (string) - id for the VC in vault
+   * @param storageRegion (string) - (optional) specify region where credentials will be stored
    * @returns a single VC
    */
-  async getCredentialByIndex(credentialIndex: number): Promise<any> {
-    return this._walletStorageService.getCredentialByIndex(credentialIndex)
+  async getCredentialById(credentialId: string, storageRegion?: string): Promise<any> {
+    return this._walletStorageService.getCredentialById(credentialId, storageRegion)
   }
 
   /**
-   * @description Delete credential by id if found in given range
-   * @param id - id of the credential
-   * @param credentialIndex - credential to remove
+   * @description Retrieve all credential
+   * @param types (string[][]) - (optional) specify credential types to filter
+   * @param storageRegion (string) - (optional) specify region where credentials will be stored
+   * @returns a single VC
+   */
+  async getAllCredentials(types?: string[][], storageRegion?: string): Promise<any> {
```

Comment:
Same suggestion here, `getAllCredentials` should not have `types` as its parameter.
If we need a method that accepts types (do we need it?), it should be called `getCredentialsByTypes`, while `getAllCredentials` should return _all_ credentials.